arcade/utils.py

```diff
@@ -17,8 +17,16 @@ class PerformanceWarning(Warning):
     pass
 
 
-def warning(message: str, warning_type: Type[Warning]):
+class NameChangeWarning(Warning):
```

Comment:
As you said the built-in one gets filtered, thats why a custom warning. I don't think `ReplacementWarning` kind of sounds right, like it kind of means that we have changed the functionality too, I guess? Although if you, I or anyone can't think of a better name we will kind of go with `ReplacementWarning` or just won't change name.

Reply:
> like it kind of means that we have changed the functionality too

I think we have since the argument order is different and passing the old one will raise exceptions.

Reply:
Oh right 👍🏽

Reply:
See the new reviews, btw. This point will be even truer after the exception type is changed for the new functions.

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1650/commits/0af3ba990f9eacd0a13e188aa300615db4226460